main_server/semantic/bmlt_semantic.js

```diff
@@ -981,6 +986,18 @@ BMLTSemantic.prototype.createServiceBodyList = function (
     };
 };
 
+/*******************************************************************************************/
+/**
+ \brief Sort By Key
+ */
+/*******************************************************************************************/
+BMLTSemantic.prototype.sortByKey = function sortByKey(array, key) {
```

Comment:
You don’t have to name the function twice here. You can just assign sortByKey to function(...) { ... }

Reply:
oh yes thanks, a little redundant eh